src/Views/Format.elm

```diff
@@ -70,7 +70,10 @@ formatFloat decimals float =
             FormatNumber.format { frenchLocale | decimals = Exact dc }
                 >> String.replace "−" "-"
     in
-    if float == 0 then
+    if isNaN float then
+        "NaN"
```

Comment:
I'm super reluctant to display "NaN" as it's highly associated with poor code. I really insist we rather display a red "Error" string rather than NaN.

Reply:
And while we're at testing if the score is `NaN`, we could as well render an explicit error message so users may have something to copy and paste when reporting the issue, which is likely to help maintainers a lot on the long run.

Reply:
AND, sorry to insist a lot, but we MUST at least file a tech-debt card to make this `getTransportBetween` fn returning a `Result` so we don't have to mess around with fake error results.

Reply:
AND sorry to be a pain in the butt.

Reply:
I agree returning a Result really is the better solution. This is option A.

For now in the short term we have the choice between :
- option B, the current proposal which is propagating a NaN up to display, without much harm (I've checked with bad data).
- option C, the current state, which is breaking the runtime with a weird behaviour when the distance data is not complete.

Which option do you prefer in the short term?

Reply:
At least replace the `"NaN"` text with either `"Err"` or `"---"`. I don't want `NaN` to be printed on the web page.

Reply:
btw I created the card I suggested so you don't have to do it https://www.notion.so/Afficher-une-erreur-l-utilisateur-en-cas-de-distance-entre-deux-pays-inconnue-e3866a56500d4dfd9c2fc1d3d44c58a1